Makefile

```diff
@@ -129,7 +132,9 @@ go-openapi: $(OPENAPI_OUT_DIR)/backend/types.gen.go \
 			$(OPENAPI_OUT_DIR)/workflows/steps/common/repo_downloader/server.gen.go
 
 clean-go-openapi:
-	rm -rf $(addprefix $(OPENAPI_OUT_DIR)/, */types.gen.go */server.gen.go)
+	rm -rf $(addprefix $(OPENAPI_OUT_DIR)/backend, */types.gen.go */server.gen.go)
+	rm -rf $(addprefix $(OPENAPI_OUT_DIR)/workflows/steps/web_feature_consumer, */types.gen.go */server.gen.go)
+	rm -rf $(addprefix $(OPENAPI_OUT_DIR)/workflows/steps/common/repo_downloader, */types.gen.go */server.gen.go)
```

Comment:
Is adding the wildcard '*' important here? AFAICS the paths would be correct without it.

Reply:
Good point! Fixed